### ISSUE TITLE
Chat message roars do not play until the player is on-screen

### ISSUE THREAD

**Issue by Souretsu** [TheIsleGame/bugs#508]
## Expected Behavior

The noise caused by typing in chat should play as soon as the message is sent.

## Current Behavior

If someone off-camera speaks in chat, no roar will play until they appear on-screen. Animation also does not play until the sound does so.

## Steps to reproduce

Requires at least 2 people.

1) Spawn in as the same dinosaur species (not required but useful for confirmation)
2) Have Player 2 walk out of view for Player 1.
3) Have Player 2 speak in chat.
4) Wait any amount of time, then have Player 1 pan camera over to Player 2 (or have Player 2 walk into view of Player 1). Observe

Video: https://youtu.be/aXBrLGB9zBk

## Build Information

> **Game Version:** 0.1.51.9 (dev)

> **Operating System**: Windows

> **Network Role**: Client

**Comment by Hypno123** [comment]
This is a duplicate of #478.

I will however add the video from here to the comments of that report.

**Comment by Souretsu** [comment]
Can confirm that the player in the video did not make any calls aside from the ones generated by using the chat.

Upon further testing on the Official Testing Server, this odd delay does not happen with only 2-3 people on the server.  Have not tested other populations but this leads me to believe that it's possibly related to the server load? In my uneducated opinion at least.

**Comment by looterz** [comment]
Duplicate of #478

**Comment by dinosauriac** [comment]
You sure this is a duplicate? Has anyone checked if the same thing happens with the voice commands? The previous problem was some player being unable to hear calls unless you're looking at the speaker, but didn't seem to have this component where the animation would play delayed when they finally come into your view.

**Comment by looterz** [comment]
You're right, at first glance it looked like a duplicate of that issue, but this may be another engine bug all together. I will re-open this issue for separate tracking.
